player/command.c

```diff
@@ -5736,6 +5736,10 @@ static void cmd_track_reload(void *p)
     }
 
     struct track *nt = mpctx->tracks[nt_num];
+
+    if (!nt->lang)
+        nt->lang = mp_guess_lang_from_filename(nt, nt->external_filename);
```

Comment:
Shouldn't this only be done if it's external and has an external filename?

Reply:
The function returns early in that case because `nt_num` is -1.

Reply:
Ah okay, it wasn't obvious to me at a glance.